.github/workflows/contributors-readme.yml

```
@@ -0,0 +1,24 @@
+        paths-ignore:
+          - 'README.md'
+
+jobs:
+    contrib-readme-job:
+        runs-on: ubuntu-latest
+        name: Update files
+        permissions:
+          contents: write
+          pull-requests: write
+        steps:
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)